bin/rucio

```diff
@@ -444,16 +445,24 @@ def list_file_replicas(args):
                             for pfn in replica['rses'][rse]:
                                 print(pfn)
         else:
+            if args.all_states:
+                header = ['SCOPE', 'NAME', 'FILESIZE', 'ADLER32', '(STATE) RSE: REPLICA']
+            else:
+                header = ['SCOPE', 'NAME', 'FILESIZE', 'ADLER32', 'RSE: REPLICA']
             for replica in replicas:
                 if 'bytes' in replica:
                     for rse in replica['rses']:
                         for pfn in replica['rses'][rse]:
+                            if args.all_states:
+                                rse_string = '({2}) {0} {1}'.format(rse, pfn, ReplicaState[replica['states'][rse]].value)
```

Comment:
Was the colon after the RSE name removed on purpose?

Reply:
It is a mistake, I correct it now, sorry